client/verta/verta/_tracking/experimentrun.py

```diff
@@ -1012,38 +1015,6 @@ def log_dataset_version(self, key, dataset_version, overwrite=False):
 
         self._clear_cache()
 
-    def log_dataset_path(self, key, path):
-        """
-        Logs the filesystem path of an dataset to this Experiment Run.
-
-        .. deprecated:: 0.13.0
-           The :meth:`~ExperimentRun.log_dataset_path` method will removed in v0.16.0; consider using
-           :meth:`client.set_dataset(…, type="local") <verta.client.Client.set_dataset>` and :meth:`~ExperimentRun.log_dataset_version` instead.
-
-        This function makes no attempt to open a file at `dataset_path`. Only the path string itself
-        is logged.
-
-        Parameters
-        ----------
-        key : str
-            Name of the dataset.
-        dataset_path : str
-            Filesystem path of the dataset.
-
-        """
-        _utils.validate_flat_key(key)
-
-        warnings.warn("`log_dataset_path()` is deprecated and will removed in a later version;"
-                      " consider using `client.set_dataset(..., type=\"local\")`"
-                      " and `run.log_dataset_version()` instead",
-                      category=FutureWarning)
-
-        # create impromptu DatasetVersion
-        dataset = _dataset.LocalDataset(self._conn, self._conf, name=key)
-        dataset_version = dataset.create_version(path=path)
```

Comment:
This is extremely hacky to implement in the new system, and this method has been deprecated for a while anyway, so 👋